cms/src/main/java/uk/nhs/digital/apispecs/module/ApiSpecSyncFromApigeeModule.java

```diff
@@ -39,22 +61,23 @@
         // job here rather than in doInitialize which is only being called on system startup.
 
         synchronized (configurationLock) {
+            for (final String jobName: JOB_NAMES) {
+                try {
+                    log.info("Configuring job {}/{} for synchronising API Specifications from Apigee.", JOB_GROUP, jobName); // the only one called on the config node change
```

Comment:
Log content probably wants to be more generic now that we have added a job that is _not_ about synchronising from Apigee. How about something like?:

> Configuring job {}/{} for updating API specifications.